docs/content/modeling/public-access.mdx

```diff
@@ -115,18 +115,31 @@ Let us create a relationship tuple that states: **any user can view document:com
   ]}
 />
 
+:::caution Wildcard syntax usage
+Please note that `*` is a special <ProductName format={ProductNameFormat.ShortForm}/> syntax meaning **everyone** when used as a _user_ within a _relationship tuple_. It is not a wildcard or regex expression.
+
+**You cannot use the `*` syntax in the tuple's object field.**
+
+The following syntax is invalid:
+<WriteRequestViewer
+  relationshipTuples={[
+    {
+      _description: 'Not allow to specify * in the object field',
+      user: 'user:*',
+      relation: 'view',
```

Comment:
```suggestion
      relation: 'can_view',
```

Reply:
We should use relation `view` as I want the example to relate to the model described in the page https://openfga.dev/docs/modeling/public-access